Documentation/signing-and-verification-guide.md

```diff
@@ -0,0 +1,333 @@
+https://example.com/images/example.com/hello-0.0.1-linux-amd64.aci
+https://example.com/images/example.com/hello-0.0.1-linux-amd64.sig
+```
+
+The first response contains the template URL used to download the ACI image and detached signature file.
+
+```
+<meta name="ac-discovery" content="example.com/hello https://example.com/images/{name}-{version}-{os}-{arch}.{ext}">
+```
+
+Rocket populates the `{os}` and `{arch}` based on the current running system.
+The `{version}` will be taken from the tag given on the command line or "latest" if not supplied.
+The `{ext}` will be substituted appropriately depending on artifact being retrieved: .aci will be used for ACI images and .sig will be used for detached signatures.
+
+Once the ACI image has been downloaded rocket will extract the image's name from the image metadata. The image's name will be used to locate trusted public keys in the rocket keystore and perform signature validation.
+
+## Verifying Images with Rocket
+
+### Establishing Trust
+
+By default rocket does not trust any signing keys. Trust is established by storing public keys in the rocket keystore.
+The following directories make up the default rocket keystore layout:
+
+```
+/etc/rkt/trustedkeys/root.d
+/etc/rkt/trustedkeys/prefix.d
+/usr/lib/rkt/trustedkeys/root.d
+/usr/lib/rkt/trustedkeys/prefix.d
+```
+
+System administrators should store trusted keys under `/etc/rkt` as `/usr/lib/rkt` is designed to be used by the OS distribution.
+Trusted keys are saved in the desired directory named after the fingerprint of the public key.
+System administrators can "disable" a trusted key by writing an empty file under `/etc/rkt`.
+For example, if your OS distribution shipped with the following trusted key:
+
+```
+/usr/lib/rkt/trustedkeys/prefix.d/coreos.com/a175e31de7e3c5b9d2c4603e4dfb22bf75ef7a23
+```
+
+you can disable it by writing the following empty file:
+
+```
+/etc/rkt/trustedkeys/prefix.d/coreos.com/a175e31de7e3c5b9d2c4603e4dfb22bf75ef7a23
+```
+
+### Trusting the example.com/hello key
+
+#### Download the public key
+
+```
+$ curl -O https://example.com/pubkeys.gpg
+```
+
+#### Capture the public key fingerprint
+
+```
+$ gpg --no-default-keyring --fingerprint --keyring ./pubkeys.gpg kelsey.hightower@coreos.com
+```
+
+```
+pub   2048R/26EF7A14 2015-01-09
+      Key fingerprint = B346 E31D E7E3 C6F9 D1D4  603F 4DFB 61BF 26EF 7A14
+uid       [ unknown] Kelsey Hightower (ACI signing key) <kelsey.hightower@coreos.com>
+sub   2048R/B9C074CD 2015-01-09
+```
+
+Remove white spaces and convert to lowercase:
+
+```
+$ echo "B346 E31D E7E3 C6F9 D1D4  603F 4DFB 61BF 26EF 7A14" | \
+  tr -d "[:space:]" | tr '[:upper:][:lower:]' '[:lower:][:upper:]'
+```
+```
+b346e31de7e3c6f9d1d4603f4dfb61bf26ef7a14
+```
+
+#### Trust the key for the example.com/hello prefix
+
+```
+mkdir -p /etc/rkt/trustedkeys/prefix.d/example.com/hello
+mv pubkeys.gpg  /etc/rkt/trustedkeys/prefix.d/example.com/hello/b346e31de7e3c6f9d1d4603f4dfb61bf26ef7a14
```

Comment:
I was thinking that a `rkt trust` command could do this for you, including the fingerprint/rename step. A flag like `rkr trust --prefix coreos.com/etcd` or by default trust it globally.